packages/ceramic-core/src/ceramic.ts

```diff
@@ -234,9 +238,7 @@ class Ceramic implements CeramicApi {
    * @param genesisCid
    */
   getDocFromMap(genesisCid: any): Document {
-    const id = DoctypeUtils.createDocIdFromGenesis(genesisCid)
-    const normalizedDocId = DoctypeUtils.normalizeDocId(id)
-    return this._docmap[normalizedDocId]
+    return this._docmap[genesisCid.toString()]
```

Comment:
I assume this should have type CID and we should convert it to a docid before returning the map?

Reply:
yeah the problem is getting the type, you have dag.get instead of just hit cached object

but yeah inconsistent with every where else, would almost leans towards cid, but that would probably make code less readable everywhere else

Reply:
but docIDs are used elsewhere in this map?